source/upgrade.txt

```diff
@@ -70,12 +70,12 @@ The breaking changes in this section are categorized by the major
 version releases that introduced them. When upgrading driver versions,
 address all the breaking changes between your current version and the
 planned upgrade version. For example, if you are upgrading the driver
-from v3.x to v5.x, address all breaking changes listed under v4.x and
-v5.x.
+from v3.x to v5.x, address all breaking changes listed under v4.0 and
+v5.0.
 
-.. _node-breaking-changes-v6.x:
+.. _node-breaking-changes-v6.0:
```

Comment:
S: instead of replacing the anchor, add a new one underneath the old one so that we dont potentially break any links. Applies to all sections!